dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java

```diff
@@ -220,16 +221,17 @@ public void doSaveProperties(long version) {
                 } finally {
                     lock.release();
                 }
-            }
-        } catch (Throwable e) {
-            savePropertiesRetryTimes.incrementAndGet();
-            if (savePropertiesRetryTimes.get() >= MAX_RETRY_TIMES_SAVE_PROPERTIES) {
+            } catch (Throwable e) {
                 if (e instanceof OverlappingFileLockException) {
-                    // fix #9341, ignore OverlappingFileLockException
                     logger.info("Failed to save registry cache file for file overlapping lock exception, file name " + file.getName());
                 } else {
-                    logger.warn("Failed to save registry cache file after retrying " + MAX_RETRY_TIMES_SAVE_PROPERTIES + " times, cause: " + e.getMessage(), e);
+                    throw e;
                 }
+            }
```

Comment:
`OverlappingFileLockException` still needs to be retried.

Reply:
Thanks for your review